tools/collapse_schemas/collapse-schemas.sh

```diff
@@ -0,0 +1,104 @@
+#!/bin/bash
+# The purpose of this script is a one time data migration from all of our 128
+# schemas into the public schema.
+
+set -euo pipefail
+
+usage() {
+  echo "Usage: collapse-schemas.sh <SERVER> [migrate]"
+  echo "Valid servers are: jobsrv, originsrv, sessionsrv"
+  exit 1
+}
+
+if [ -z "${PGPASSWORD:-}" ]; then
+  echo "Please ensure that PGPASSWORD is exported in your environment and run this script again."
+  exit 1
+fi
+
+execute_sql() {
+  local sql="$1"
+  local server="$2"
+
+  hab pkg exec core/postgresql psql -t -U hab -h 127.0.0.1 -p 5433 -c "$sql" "builder_$server"
+}
+
+process_shards() {
```

Comment:
One more thing to think about - since this script is not idempotent, maybe check here to see if the migration has already been run (check the flag) and print a nice message and exit immediately if so.

Reply:
Yeah, good call